src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -278,6 +283,14 @@ def get_obs_data(bufr, profile_meta_data, add_qc, record_number=None):
         for k in obs_data.keys():
             obs_data[k] = obs_data[k][good]
 
+    if addLSW:
+        bangid = bang == 0
+        bang[bangid] = float_missing_value
+        bang_err[bangid] = float_missing_value
+
+        lsw = bang_err/bang * 100   # CDACC saves LSW in radians as bang_err. Convert to percentage
+        lsw[bang == float_missing_value] = float_missing_value
+        obs_data[('localSpectralWidth', "MetaData")] = assign_values(lsw)
```

Comment:
Since this is normalized by observation and the unit is %, perhaps localSpectralWidth does not reflect what it is. Any name convention? @huishao-r  probably has a better idea.